makefile

```diff
@@ -213,7 +224,11 @@ doccoverage:
 	@echo '$@ done.'
 
 .PHONY: check
-check: pylint.log
+check: pylint.log flake8.log
+	@echo '$@ done.'
+
+.PHONY: flake8
+flake8: flake8.log
 	@echo '$@ done.'
```

Comment:
If you really thing we should have a `flake8` virtuial target in addition to the `check` target, then please add it to the `help` target.